chunky/src/java/se/llbit/chunky/map/WorldMapLoader.java

```diff
@@ -58,7 +58,7 @@ public WorldMapLoader(ChunkyFxController controller, MapView mapView) {
     RegionChangeWatcher regionWatcher = new RegionChangeWatcher(this, mapView);
 
     // Start worker threads.
-    RegionParser[] regionParsers = new RegionParser[Integer.parseInt(System.getProperty("chunky.mapLoaderThreads", "3"))];
+    RegionParser[] regionParsers = new RegionParser[Integer.parseInt(System.getProperty("chunky.mapLoaderThreads", "12"))];
```

Comment:
This number is (and was) pretty arbitrary. Should we use the thread count from `Settings` by default?

Reply:
commiting this was entirely accidental and is justt what I use to make my debugging faster, Yeah I think that'd be best